cmd/list/instancetypes/cmd.go

```diff
@@ -78,10 +76,15 @@ func run(cmd *cobra.Command, _ []string) {
 	fmt.Fprintf(writer, "ID\tCATEGORY\tCPU_CORES\tMEMORY\t\n")
 
 	for _, machine := range machineTypes {
+		if !machine.Available {
```

Comment:
Previously this command was a way to see all supported types, right?
Now typef you dont have quota for will just disappear.
Maybe it'd be more useful to list them all with a new QUOTA column? (ideally not just true/false but a number)

Reply:
Hmm, this is tricky if most types are not limited by OCM quota...
Saying "unlimited" may be a bit confusing too, because they are limited by your AWS account.